src/BlazorWasmSdk/Tasks/Microsoft.NET.Sdk.BlazorWebAssembly.Tasks.csproj

```diff
@@ -1,35 +1,54 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project>
 
   <PropertyGroup>
     <BlazorWasmSdkRoot>$(RepoRoot)\src\BlazorWasmSdk\</BlazorWasmSdkRoot>
     <PackageId>Microsoft.NET.Sdk.BlazorWebAssembly</PackageId>
     <OutDirName>$(Configuration)\Sdks\$(PackageId)\tools</OutDirName>
   </PropertyGroup>
 
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
```

Comment:
Could you add explain why this is done this way? Also if you could add a comment to that effect that would be great

Reply:
I guess it's a common theme across this repo. Let's ignore this.

Reply:
some of the tests are "exported" as a dotnet tools. These tests are not, so we don't need the extra complication.